tests/test_fields.py

```diff
@@ -672,6 +673,7 @@ def test_valid_inputs(self, *args):
             assert self.field.run_validation(input_value) == expected_output, \
                 'input value: {}'.format(repr(input_value))
 
+    @pytest.mark.xfail(pytz and DJANGO_VERSION >= (5, 0), reason="Django v5 drops support for pytz")
```

Comment:
This looks like it'll skip a whole load of tests regardless of if `pytz` is relevant to that particular test case or not.